graphdatascience/aura_graph_data_science.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, Optional, Tuple
+
+from neo4j import GraphDatabase
+from pandas import DataFrame
+
+from .query_runner.neo4j_query_runner import Neo4jQueryRunner
+from .server_version.server_version import ServerVersion
+from graphdatascience import GraphDataScience
+from graphdatascience.query_runner.aura_db_arrow_query_runner import (
+    AuraDbArrowQueryRunner,
+    AuraDbConnectionInfo,
+)
+
+
+class AuraGraphDataScience(GraphDataScience):
```

Comment:
We talked about the set_database method yesterday. Should we also change the semantics here?

Reply:
What exactly do you mean here? the `set_database` and `set_bookmark` calls are delegated to the `AuraDbQueryRunner` here which then delegates to an `_db_query_runner`.